### DISCUSSION TITLE
How to force controller input when not using big picture mode ?

### DISCUSSION THREAD

**Discussion by taker1988** [LizardByte/community#521] [Sunshine]
### Select Topic Area

Question

### Body

Hi,
I use Linux Mint 22 and the latest Sunshine and Moonshine version. (no flatpak)
Moonshine runs on my ShieldTV.

With Windows 11 everything runs perfectly, but with Linux I have problems.
As I read, there are problems with Linux and the Steam Big Picture mode. Everything is very, very laggy. So I tried to integrate my Steam games directly into Sunshine, which works.

However, I now have the problem that my X-Box Series X controller connected to the Shield is not recognized, or not for every game. With some it works.

Is there a command that I can add to Sunshine to force the controller input?

I think the problem is that I start the game via steam directly and without the big picture mode. But I am not sure.

I hope someone can help me or has an idea how I could do this. :)

Thank you very much

**Comment by willow-sprite** [comment]
I actually have a direct solution for once.

a) launch desktop mode
b) make sure steam is closed and not running in background
c) launch steam and hold down a button on the d-pad while it opens, this will make sure steam grabs the controller.

steam will now emulate a controller until moonlight disconnects.

... and will not automatically reconnect if moonlight disconnects, so you will have to close your game and steam if you accidentally disconnect.

[a comment by an automated account is not shown]
